app.js

```diff
@@ -1,9 +1,19 @@
 const express = require("express");
 const apiRouter = require("./routes/api-router");
-
+const result = dotenv.config();
```

Comment:
This line needs to be right at the top of the file, before any other `require` statements (also says this on the `dotenv` documentation). The reason for this is because the other libraries you include after, may need the custom values that you specified in your `.env` file.